mopidy/core/playback.py

```diff
@@ -560,3 +572,26 @@ def _trigger_seeked(self, time_position):
         # TODO: Trigger this from audio events?
         logger.debug('Triggering seeked event')
         listener.CoreListener.send('seeked', time_position=time_position)
+
+    def _export_state(self):
+        """Internal method for :class:`mopidy.Core`."""
+        return models.PlaybackState(
+            tlid=self.get_current_tlid(),
+            position=self.get_time_position(),
```

Comment:
I think `position` should be changed to `time_position` to match the naming in the core API and also to not be confused with e.g. tracklist position.